addons/mail/models/mail_thread.py

```diff
@@ -1940,7 +1940,7 @@ def _message_compose_with_view(self, views_or_xmlid, message_log=False, **kwargs
 
     def message_post_with_view(self, views_or_xmlid, **kwargs):
         """ Helper method to send a mail / post a message using a view_id """
-        self._message_compose_with_view(views_or_xmlid, **kwargs)
+        self.with_context(inherit_branding=False)._message_compose_with_view(views_or_xmlid, **kwargs)
```

Comment:
This should move into website as an override, as it has no usage in base mail module.

Reply:
Done :+1: